### PR TITLE
Add a Get method (Get all users):

### PR DESCRIPTION
**Pull request by Danie8**
Service interface declaration in ServiceApplication.Java
Service implementation (Override) in ServiceApplicationimpl.Java
Use service declaration in ControllerAplication.java

### CODE REVIEW

**Review by rodrigo-velasco — CHANGES_REQUESTED** [review]
The data in the application.properties file should not be committed.

File: src/main/java/com/agilethought/internship/sso/services/ServiceApplication.java [hunk 1]
```diff
@@ -6,5 +6,6 @@
 public interface ServiceApplication {
 
 	public UserId createUser(User user);
+	public java.util.List<User> getUsers();
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/agilethought/internship/sso/services/ServiceApplication.java]: Please add "java.util.List" like a import, the method should be like: "public List<User> getUsers();"

File: src/main/java/com/agilethought/internship/sso/services/ServiceApplicationimpl.java [hunk 2]
```diff
@@ -24,6 +26,11 @@ public UserId createUser(User user) {
 		userId.setId(userIdDb);
 		return userId;
 	}
-
 	
+	@Override
+	public java.util.List<User> getUsers() {
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/agilethought/internship/sso/services/ServiceApplicationimpl.java]: Same scenario here, please add "Please add "java.util.List" like a import.

File: src/main/java/com/agilethought/internship/sso/controller/ControllerAplication.java [hunk 3]
```diff
@@ -48,4 +51,11 @@ public UserId createUser(@RequestBody User user) {
 		UserId userId = serviceApplication.createUser(user);
 		return userId;
 	}
+	
+	@GetMapping(value="/at-sso-api/api/v1/user", produces = "application/json")
+	@ResponseStatus(value = HttpStatus.OK)
+	public java.util.List<User> getAllUsers(){
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/agilethought/internship/sso/controller/ControllerAplication.java]: Please add "java.util.List" like a import, the method should be like: "public List<User> getAllUsers(){"

File: src/main/java/com/agilethought/internship/sso/services/ServiceApplicationimpl.java [hunk 4]
```diff
@@ -24,6 +26,11 @@ public UserId createUser(User user) {
 		userId.setId(userIdDb);
 		return userId;
 	}
-
 	
+	@Override
+	public java.util.List<User> getUsers() {
+		java.util.List<User> response=repositoryApplication.findAll();
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/agilethought/internship/sso/services/ServiceApplicationimpl.java]: Same scenario with this response list object, please add "Please add "java.util.List" like a import.

**Review by VictorMunguiaEnciso — CHANGES_REQUESTED** [review]

**Review by VictorMunguiaEnciso — APPROVED** [review]

**Review by rodrigo-velasco — APPROVED** [review]

**Pull request merged**